lib/blacklight/xml_parser.rb

```diff
@@ -113,7 +113,8 @@ def self.get_master_parent(pre_data, parents_ids, parent_id)
   # non-metadata file.
   ##
   def self.iterate_files(zipfile)
-    files = zipfile.glob("*/**/**")
+    files = zipfile.entries.select(&:file?)
```

Comment:
That is really interesting. When I put a byebug in the actual method and then run `rake blacklight:imscc` on the export titled ExportFile_SYS_196_20161028072839 I get: 
```
zipfile.glob("**/**").select(&:directory?).size
=> 91
zipfile.glob("**/**").select(&:file?).size
=> 1660
zipfile.entries.count
=> 1751
```